docs/UserGuide.md

```diff
@@ -30,12 +30,14 @@ Head over to [Quick Start](#2-quick-start) or [Features](#4-features) for more i
 
 ### 1.2 How to use the user guide
 
-As most applications in the market are not CLI-based, this may come across 
-as new to some users. We strongly recommend that first time users read through the user guide to gain a basic understanding
+As most applications on the market are not CLI-based, this may come across 
+as new to some users. We strongly recommend that first-time users read through the user guide to gain a basic understanding
 of how to use our app. We have also included links to navigate around the user guide to allow easier access
 of related information. Some tips and tricks, as well as cautions are included throughout the guide to
-help you get more out of the application and avoid certain common dangers. Lastly, please do use the table of contents
-at the top of the document for easier navigation throughout document. 
+help you get more out of our application and avoid certain common dangers. Also, links back to the top of the page
+like below are present at the end of every section. To allow for easier navigation 
```

Comment:
For easier navigation around this User Guide, we suggest using our table of contents together with these links to quickly find and access the information you need.

Reply:
Noted and amended